test/sequential/test-worker-prof.js

```diff
@@ -19,20 +20,25 @@ if (process.argv[2] === 'child') {
   }
   const pingpong = `
   let counter = 0;
+  const fs = require('fs');
   const { Worker, parentPort  } = require('worker_threads');
   parentPort.on('message', (m) => {
-    if (counter++ === 1024)
+    if (counter++ === 10)
       process.exit(0);
      parentPort.postMessage(
-       m.toString().split('').reverse().toString().replace(/,/g, ''));
+       fs.readFileSync(m.toString()).toString());
   });
   `;
 
   const { Worker } = require('worker_threads');
-  const data = 'x'.repeat(1024);
   const w = new Worker(pingpong, { eval: true });
   w.on('message', (m) => {
-    w.postMessage(m.toString().split('').reverse().toString().replace(/,/g, ''));
+    w.postMessage(process.execPath);
+  });
+
+  w.on('error', (e) => {
+    console.error(`worker exited with error: ${e}`);
+    process.exit(1);
   });
```

Comment:
```suggestion
```

Uncaught `error` events exit the process with a better message, see https://github.com/nodejs/node/pull/32245#pullrequestreview-374101243